### ISSUE TITLE
[Question] Fallback on single-line comment

### ISSUE THREAD

**Issue by philipGaudreau** [numToStr/Comment.nvim#112]
Hi,

Is there a way to fallback on single-line comment (perhaps with pre-hook?) when invoking blockwise comment when it is not defined for a particular language.

For example, in python, multi-line comments do not exist, so calling blockwise comment on a region in visual mode will result in only the first line being highlighted.

Thanks!

**Comment by numToStr** [comment]
Yes you can fallback with `pre_hook`. But the example you made will do the same in either way.
